common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -790,17 +851,44 @@
             *     out of the tabbing order.
             *
             */
-            addPaddings: function () {
+            addPaddings: function() {
+                var topSpacer = HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML([
+                            '<li class="spacing" style="height: {height}px">',
+                                '<a href="#transcript-end-{id}" id="transcript-start-{id}" class="transcript-start"></a>', // jshint ignore:line
+                            '</li>'
+                        ].join('')),
+                        {
+                            id: this.state.id,
+                            height: this.topSpacingHeight()
+                        }
+                    ).toString();
+
+                var bottomSpacer = HtmlUtils.interpolateHtml(
+                        HtmlUtils.HTML([
+                            '<li class="spacing" style="height: {height}px">',
+                                '<a href="#transcript-start-{id}" id="transcript-end-{id}" class="transcript-end"></a>', // jshint ignore:line
+                            '</li>'
+                        ].join('')),
+                        {
+                            id: this.state.id,
+                            height: this.bottomSpacingHeight()
+                        }
+                    ).toString();
 
-                this.subtitlesMenuEl
-                    .prepend(
-                        $('<li class="spacing"><a href="#transcript-end-' + this.state.id + '" id="transcript-start-' + this.state.id + '" class="transcript-start"></a>') // jshint ignore: line
-                            .height(this.topSpacingHeight())
+                HtmlUtils.prepend(
+                    this.subtitlesMenuEl,
+                    HtmlUtils.HTML(
```

Comment:
`HtmlUtils.prepend()` is HtmlSnippet aware.  So let `topSpacer` above stay an HtmlSnippet by not calling `toString()`, and then pass it in here without wrapping it in `HtmlUtils.HTML()`.  This applies to `bottomSpacer` and several other places in this file.  Once you use HtmlUtils to create something, it is usually an HtmlSnippet, and you can continue to use it in later HtmlUtils calls with it just working appropriately.